docs/conf/eu/2017/cfp.rst

```diff
@@ -3,5 +3,113 @@
 Call for proposals
 ==================
 
-We're still ironing out the call for proposals for our North America conference in Portland,
-stay tuned for updates to this page as we get ready to launch the CFP for Europe!
+Hello hello, fellow documentarians! It's that time of year again: We’re very
```

Comment:
Just a side note, I think it's kinda funny how the best way to figure out whether Sam or I wrote copy is by the "Hello hello" vs "Greetings" intro. <3